### ISSUE TITLE
User profile page - change profile picture

### ISSUE THREAD

**Issue by Saul-Caban-UPRM** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#218]
**Objective:**
Adding functionality to change profile picture found in the future user profile
**Task Description:**
This just allows the user to change their profile to their liking. providing a more user-interaction environment 
**Implementation:**
Create querries saving the user image in the database and create a method to retrieve said image from our database by going into our user.js files found in controller and dao folders. Next is displaying said image onto the homepage by saving the image into a variable and formatting the image in the "home.ejs "file. 
**Testing and Debugging:**
For this task, checking that the picture they chose is on the home page displayed should tell us if it's working or not. 
**Deadline:**
April 20, 2024

**Comment by evnd04** [comment]
Remember when possible to add the implementation when you find a clear way to implement the feature.

**Comment by Saul-Caban-UPRM** [comment]
The implementation has been update. I would also like to add that @Javi510348  and I had to implement other parts for this task. For example the way the image is stored it has to go through a folder that its created in our project and save the image to the database. Then we have to make sure that the picture is deleted locally. We also added page redirection as soon you click on the upload photo.
